doc/intro.rst

```diff
@@ -7,6 +7,8 @@ adventurelib.
 Starting a project
 ------------------
 
+Your text adventure will be run from a single python file. Create a new plain text file (we'll call it my_game.py) and open it in your favorite editor. 
```

Comment:
Adventurelib doesn't require that games run from a single file, so perhaps you could clarify that this applies for this example.